third_party/open-policy-agent/gatekeeper/helmify/static/values.yaml

```
@@ -0,0 +1,48 @@
+image:
```

```suggestion
controllerManager: 
	image:
```

let's move this one level below